.travis.yml

```diff
@@ -1,28 +1,13 @@
-dist: trusty
 language: python
 python:
-  - 2.7
-  - 3.5
-  - 3.6
-addons:
-  apt:
-    packages:
-    - libatlas-dev
-    - libatlas-base-dev
-    - liblapack-dev
-    - gfortran
-    - libmpfr-dev
-    - libhdf5-dev
+  - "2.7"
+  - "3.6"
+  - "3.7"
 cache: pip
 install:
-  - pip install 'matplotlib==1.5.2' tox
-  - pip install .
```

Comment:
I'm wondering if this line is required for the deployement stage ?

Reply:
iirc, pinning the matplotlib version was to make sure the tests passed